src/scripts/lib/db.js

```diff
@@ -153,14 +149,27 @@ export default class Db {
     this.set(this.togglButton.$user.id + '-defaultProjects', {});
   }
 
-  get (key, defaultValue) {
-    const hasDefaultValue = typeof defaultValue !== 'undefined';
-    return browser.storage.sync.get(hasDefaultValue ? { [key]: defaultValue } : key)
+  get (key) {
+    const hasDefaultValue = typeof DEFAULT_SETTINGS[key] !== 'undefined';
+    const defaultValue = DEFAULT_SETTINGS[key];
```

Comment:
Ultra-minor nitpick but you could swap L153 and L154 and use `defaultValue` in the `typeof` check